package.json

```diff
@@ -212,7 +212,7 @@
   },
   "engines": {
     "node": "^16.13.0",
-    "npm": ">=8.0.0"
+    "npm": "^9.0.0"
```

Comment:
I've switched this to a caret. As we can see from this Pr, jumping a major NPM version tends to require some intervention/review from us